WordPress/src/main/java/org/wordpress/android/viewmodel/pages/SearchListViewModel.kt

```diff
@@ -1,10 +1,10 @@
 package org.wordpress.android.viewmodel.pages
 
+import android.content.Context
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.Observer
 import kotlinx.coroutines.CoroutineDispatcher
-import kotlinx.coroutines.launch
```

Comment:
Thank you for fixing this. I searched through our [detekt baseline config file](https://github.com/wordpress-mobile/WordPress-Android/blob/2b352168a2a6806edbe1e6c2ed48e8d9c6a2684d/config/detekt/baseline.xml) didn't find it to be added to the `<CurrentIssues>` tag → thus couldn't understand why CI checks on PRs ignore this violation of our lint rules. Might be able to come back with an answer later 😺

Reply:
Discussed this internally and it was a bit of a surprise also for our DevOps engineers.

After looking a bit deeper into it apparently it's related to the fact that the base ViewModel imports from `kotlinx.coroutines.launch` but Detekt is not as smart as the `Android Studio IDE` to know we don't need to import from that package in the inheriting `SearchListViewModel`.